description.md

```diff
@@ -17,6 +17,86 @@ Each section details how a specific type of content is to be developed, stored a
+
+Another desired aspect is that quizzes be easily extractable and convertible between formats, as described [below](#deployment), and repurposed by other trainers.
+
+#### Deployment
+
+Each trainer uses a specific teaching rendering platform.
+Most platforms support quizzes in various formats, such as whether they are graded or ungraded, whether the answers are multiple choice or text boxes, etc.
+Out of the existing teaching platforms, a few highlights are:
+
+- [Moodle](https://moodle.org/) is a free open-source learning platform dedicated to course management.
+It supports graded assignments, quizzes, storing teaching materials and many other educational activities.
+
+- [SmileyTutor (formerly called TutorWeb)]((https://tutor-web.net/)) is another free learning platform whose highlight is that it was built around the [SmyleyCoin cryptocurrency](#cryptocurrency-rewards) as a reward for completing drills and quizzes.
+
+- [tech.io](https://tech.io/) is another quiz platform.
+More simplistic than Moodle or SmileyTutor, it allows teachers to easily create _playgrounds_, which are learning environments designed around specific subjects, made up of reading material, followed by practical exercises and quizzes.
+
+Each of these platforms supports a different syntax for uploading questions.
+To make our quizzes comply with educational platforms easily, we need to store them in a generic format that is easily representable and automatically convertible to other required formats.
+For this reason, we are storing questions in Markdown format.
+We are developing a modular and easily extendible converter that ports quizzes from Markdown to other formats.
+
+#### Storage Format
+
+We are using Markdown as a generic storage format for quizzes because it is a text-based format, so it is suitable for written questions.
+Unlike more complex formats like LaTeX, Markdown offers a much simpler syntax which makes quizzes easier to contribute to by educators and students alike.
+
+The format we propose relies on headers to represent categories such as difficulty, topics, question text, feedback.
+Each question has a title written on the first line as a a level 1 header (`#`).
+Each category is specified as a level 2 header (`##`).
+The contents of the category consist of the text beneath the header.
+For multiple-choice questions, correct answers are marked as list entries starting with `+`, while incorrect ones are marked with `-`.
+
+See the example below:
+
+```text
+# Sample Question Name
+
+##  Difficulty
+
+easy
+
+## Tags
+
+- topic-1
+
+- topic-2
+
+## Question Text
+
+Is this a sample question?
+
+## Question Answers
+
++ Yes
+
+- No
+
+- Maybe
+
+## Feedback
+
+The question is self-reflecting and its title is "Sample Question Name", so it is a sample question.
+```
+
+Note that correct answers are marked as list entries starting with `+`, while incorrect ones are marked with `-`.
```

Comment:
Maybe you should add a few sentences about the fact that the `#*` sections are actually keys and are not predefined, therefore you can add any kind of entry (that is why it is extensible). Also, it would be nice to talk about the the program that creates the dictionary which then can be used to output the desired format.